palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/policy/Policy.java

```diff
@@ -37,27 +37,28 @@
  * service but not needed by the rest of the palisade services. That includes
  * separating the rules that need to be applied at the resource level or the record level.
  *
- * @param <RULE_DATA_TYPE> The Java class that the rules expect the records of
- *                         data to be in the format of.
+ * @param <V> The Java class that the rules expect the records of
```

Comment:
I don't like this change as it forces you to check the JavaDocs to find out what 'V' really means rather than having some clue with 'RULE_DATA_TYPE'

Reply:
should i revert the change and mark the bug in sonarQube as wont do?

Reply:
reverted and disabled rule in sonarQube